operator/internal/manifests/openshift/service_ca.go

```diff
@@ -5,10 +5,10 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// BuildServiceCAConfigMap returns a k8s configmap for the LokiStack
+// BuildGatewayCAConfigMap returns a k8s configmap for the LokiStack
 // serviceCA configmap. This configmap is used to configure
 // the gateway and components to verify TLS certificates.
-func BuildServiceCAConfigMap(opts Options) *corev1.ConfigMap {
+func BuildGatewayCAConfigMap(opts Options) *corev1.ConfigMap {
```

Comment:
Why `BuildGatewayCAConfigMap`? This seems to be the ca-bundle used for every service.